_posts/2016-09-22-ignobel.md

```diff
@@ -0,0 +1,50 @@
+* [Searching for Mr. Hyde: A five-factor approach to characterizing “types of drunks”](http://www.tandfonline.com/doi/full/10.3109/16066359.2015.1029920)
+* [A psychological formula for explaining the 'missing socks phenomenon'](http://mentalfloss.com/article/83809/can-washing-machines-really-eat-socks)
+* [Affective forecasting in an orangutan: predicting the hedonic outcome of novel juice mixes](http://www.ncbi.nlm.nih.gov/pubmed/27515937)
+
+
+<blockquote class="twitter-tweet" data-cards="hidden" data-lang="en"><p lang="en" dir="ltr">Must be an IgNobel prize waiting for this 5-year study to see if cats copy their owners&#39; accents.  <a href="https://t.co/q93Z4XhVs2">https://t.co/q93Z4XhVs2</a></p>&mdash; Jamie McMillan (@orchid_b) <a href="https://twitter.com/orchid_b/status/722377493223747584">April 19, 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+A good rule of thumb is that studying animals or quirky behavior is likely to yield a result that "makes people laugh, and then makes them think" ([levitating frogs](http://www.ru.nl/hfml/research/levitation/diamagnetic/), anyone?).
+But there are still plenty of wild and wacky things that just haven't been studied yet, so here are a few ideas for which I couldn't find scholarly literature or historical information.
+Perhaps some enterprising scientist would like to take a crack at these?
+And if any of these questions already have solid research-based answers that my cursory Googling overlooked, I want to hear about it!
+
+* Why do you always get into the slowest line at the grocery store?
+* Why are credit cards, IDs and business cards all the same size? [There's an international standard](https://en.wikipedia.org/wiki/ISO/IEC_7810), but how did it come about?
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">That&#39;s a research paper right there, formulating the problem &amp; comparing it with actual results. At least worth an Ignobel prize.</p>&mdash; Achintya Kambli (@achinto_k) <a href="https://twitter.com/achinto_k/status/726070905705623552">April 29, 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+* Are near-sighted people messier? Drawing on my own experience studying near-sighted people, I've wondered whether the blur of being near-sighted obscures small stuff like dirt and hair. This is evidenced especially in bathrooms, where near-sighted people often take off their glasses or it's otherwise steamy.
+* Are the values of precious metals and gemstones in any way correlated with their refractive index or other optical properties?
+Does shinier mean pricier?
+* Has anyone analyzed whether particular airplane safety videos or demonstrations are more effective than others?
```

Comment:
There was [a great video](https://www.youtube.com/watch?v=3ZzX8oFonuA) on my TAP Portugal flight the other day involving Olympic athletes, and that was amazing! Link to it?